course-definition.yml

```
@@ -478,3 +487,27 @@ stages:
 
       **Example:**
       - `('(cat) and \2') is the same as \1` should match "'cat and cat' is the same as 'cat and cat'".
+
+  - slug: "dr5"
+    name: "Search a single-line file"
+    difficulty: medium
+    marketing_md: |-
+      In this stage, you'll add support for searching the contents of a file with a single line of input.
```

```suggestion
      In this stage, you'll add support for searching the contents of a file with a single line.
```

Consistent with the sentence below